src/ProjectService.Data/ImageRepository.cs

```diff
@@ -0,0 +1,56 @@
+﻿using LT.DigitalOffice.ProjectService.Data.Interfaces;
+using LT.DigitalOffice.ProjectService.Data.Provider;
+using LT.DigitalOffice.ProjectService.Models.Db;
+using Microsoft.AspNetCore.Http;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace LT.DigitalOffice.ProjectService.Data
+{
+    public class ImageRepository : IImageRepository
+    {
+        private readonly IDataProvider _provider;
+        private readonly IHttpContextAccessor _httpContextAccessor;
+
+        public ImageRepository(
+            IDataProvider provider,
+            IHttpContextAccessor httpContextAccessor)
+        {
+            _provider = provider;
+            _httpContextAccessor = httpContextAccessor;
+        }
+
+        public bool Create(IEnumerable<DbProjectImage> images)
```

Comment:
why do you return bool. Always we return guids of new entity

Reply:
done